pdf/src/renderer/__tests__/__snapshots__/vueRenderer.spec.js.snap

```diff
@@ -10759,10 +10853,10 @@ exports[`rendering a full camp > renders the program 1`] = `
                                         "flexGrow": "0",
                                         "flexShrink": "0",
                                         "fontWeight": "semibold",
-                                        "lineHeight": "1.6",
+                                        "lineHeight": "1.4",
```

Comment:
It would be very cool if you would render the new content type in one of the tests

Reply:
Hmm, how do I replace the data with an existing database camp? What is the process here?

Reply:
you get the json which is used to render with the debugger, and then you tweak it until you have all data you want to render. (and maybe only the data you want to render).
but we can do that later